### PR TITLE
Fabo/ catching potential error when sending

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/vuex/modules/send.js [hunk 1]
```diff
@@ -200,6 +200,11 @@ export default ({ node }) => {
       // check response code
       assertOk(res)
 
+      // Sometimes we get back failed transactions, which shows only by them having a `code` property
+      if (res.code) {
+        throw new Error(`Something went wrong while sending the transaction.`)
```
Comment (jbibla) [inline app/src/renderer/vuex/modules/send.js]: any way we can be more specific?

Comment (fedekunze) [reply app/src/renderer/vuex/modules/send.js]: I agree we should display a better error, but we'd have to hardcode the base errors from the SDK. See https://github.com/cosmos/cosmos-sdk/issues/4013

File: app/src/renderer/vuex/modules/send.js [hunk 2]
```diff
@@ -200,6 +200,12 @@ export default ({ node }) => {
       // check response code
       assertOk(res)
 
+      // Sometimes we get back failed transactions, which shows only by them having a `code` property
+      if (res.code) {
```
Comment (sabau) [inline app/src/renderer/vuex/modules/send.js]: nice catch!

File: app/src/renderer/vuex/modules/send.js [hunk 3]
```diff
@@ -265,3 +271,44 @@ function handleSDKError(err) {
   }
   throw new Error(message)
 }
+
+function getErrorMessage(code) {
```
Comment (sabau) [inline app/src/renderer/vuex/modules/send.js]: I think would be nice to have this out of any particular module, to have it usable by others

**Pull request merged**